app/experimenter/nimbus-ui/src/components/ChangeApprovalOperations/mocks.tsx

```diff
@@ -0,0 +1,59 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import Button from "react-bootstrap/Button";
+import ChangeApprovalOperations from ".";
+import { NimbusChangeLogType, NimbusUser } from "./temp-types";
+
+export const mockUser = (email: string | null = "foo@bar.com"): NimbusUser => ({
+  email,
+});
+
+export const mockChangelog = (
+  email: string | null = "foo@bar.com",
+  message: string | null = null,
+  changedOn: DateTime | null = new Date().toISOString(),
+): NimbusChangeLogType => ({
+  changedBy: mockUser(email),
+  changedOn,
+  message,
+});
+
+export const SubjectChangeApprovalOperations = ({
+  actionDescription = "frobulate",
```

Comment:
I looked at Storybook before anything else in your PR and seeing "frobulate" everywhere made me have a nice laugh😂👌